CounterExample/ViewController.swift

```diff
@@ -41,13 +47,14 @@ extension ViewController : UITableViewDataSource {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: "CounterCell", for: indexPath) as! CounterCell
         cell.label.text = "\(mainStore.state.counters[indexPath.row])"
-        cell.action = { didIncrease in
-            if didIncrease {
+        cell.action = { actionType in
```

Comment:
Nope, I wanted to update the `CellActionType ` to that one:
```swift
enum CellActionType {
  case increase
  case decrease

  func counterActionForCell(at indexPath: IndexPath) -> Action {
    switch self {
    case .increase:
      mainStore.dispatch(CounterActionIncrease(index: indexPath.row))
    case .decrease:
      mainStore.dispatch(CounterActionDecrease(index: indexPath.row))
    }
  } 
}
```

Does it make sense to you?